src/Block/Service/AbstractClassificationBlockService.php

```diff
@@ -61,10 +61,8 @@ public function __construct(
      * @param string $field
      * @param array  $fieldOptions
      * @param array  $adminOptions
-     *
-     * @return FormBuilder
      */
-    final protected function getFormAdminType(FormMapper $formMapper, AdminInterface $admin, $formField, $field, $fieldOptions = [], $adminOptions = [])
+    final protected function getFormAdminType(FormMapper $formMapper, AdminInterface $admin, $formField, $field, $fieldOptions = [], $adminOptions = []): FormBuilderInterface
```

Comment:
We could add also parameter type declarations I guess

Reply:
I agree. I'll try to add them in next PRs.
Thank you.